release-notes/release-notes-insiders.json

```diff
@@ -1,4 +1,31 @@
 [
+    {
+        "version": "1.6.1-insiders",
+        "title": "Fiddler Everywhere v1.6.1",
+        "date": "May 27, 2021",
+        "items": [
+            {
+                "type": "Fixed",
+                "entries": [
+                    {
+                        "title": "Unable to set proxy on macOS when adapter's name has multiple spaces next to each other"
+                    },
+                    {
+                        "title": "Unable to set proxy on macOS's default Thunderbolt adapters"
+                    },
+                    {
+                        "title": "System proxy is not respected when starting the application on macOS in some cases"
```

Comment:
Use _**The** system proxy ..._ to be consistent with the next title (or the other way around)